src/srtm/srtm.jl

```diff
@@ -0,0 +1,59 @@
+
+struct SRTM <: RasterDataSource end
+
+# SRTM Mirror with 5x5 degree tiles
+const SRTM_URI = URI(scheme = "https", host = "srtm.csi.cgiar.org", path = "/wp-content/uploads/files/srtm_5x5/TIFF")
+
+resolutions(::Type{SRTM}) = ("30m",)
+defres(::Type{SRTM}) = "30m"
+
+function _raster_tile_stem(tile_index::CartesianIndex)
+    y, x = tile_index.I
+    "srtm_$(lpad(x, 2, '0'))_$(lpad(y, 2, '0'))"
+end
+
+rastername(::Type{SRTM}, tile_index::CartesianIndex) = _raster_tile_stem(tile_index) * ".tif"
+rasterpath(::Type{SRTM}) = joinpath(rasterpath(), "SRTM")
+rasterpath(T::Type{SRTM}, tile_index::CartesianIndex) = joinpath(rasterpath(T), rastername(T, tile_index))
+
+zipname(::Type{SRTM}, tile_index) = _raster_tile_stem(tile_index) * ".zip"
+zipurl(T::Type{SRTM}, tile_index) = joinpath(SRTM_URI, zipname(T, tile_index))
+zippath(T::Type{SRTM}, tile_index) = joinpath(rasterpath(T), "zips", zipname(T, tile_index))
+
+
+function getraster(T::Type{SRTM}, tile_index)
```

Comment:
The second argument to `getraster` is always `layer`. So we should do this with keywords. You could have 
`bounds=nothing, tile_index=_index_from_bounds(bounds)` or something so you can specify either the bounds or the tile.